src/test/scala/collection/CollectionTest.scala

```diff
@@ -1,3 +1,5 @@
+package collection
```

Comment:
What's the point of putting the tests into this package?

Reply:
Just to not use the empty package.